README.md

```diff
@@ -669,6 +670,7 @@ Table of Contents
   * [Contact.do](https://contact.do/) — Contact form in a link (bitly for contact forms) - totally free!
   * [debugmail.io](https://debugmail.io/) — Easy to use testing mail server for developers
   * [elasticemail.com](https://elasticemail.com) — 100 free emails/day. 1,000 emails for $0.09 through API (pay as you go).
+  * [Eva Beta](https://eva.pingutil.com) — Verify 5 million emails per day for free using REST API. No sign-ups required.
```

Comment:
no information here that it will remain free after Beta, so I'd rather not add it.

Reply:
Alright, my bad, actually the API is not in any beta testing stage but a full fleged API. If you scroll down to the bottom of its page you'll see that it is Sponsored by a company named LoginRadius and you can read the documentation of this API in their [official page](https://www.loginradius.com/blog/async/email-verification-api) which states that it has "**No hidden costs. No credit card needed.**" but doesn't mention it being in Beta stage.

*sidenote:* I should change the name to "Eva" instead of "Eva Beta"

Reply:
Yeah lets drop the `Beta` then

Reply:
Done